ui/src/App.tsx

```diff
@@ -24,7 +24,7 @@ import { ExternalLinks, Links } from "./router/links.models";
 import axios from "axios";
 
 function App() {
-  axios.defaults.baseURL = import.meta.env.VITE_WANAKU_ROUTER_BASE_URL || 'http://localhost:8080';
+  axios.defaults.baseURL = import.meta.env.VITE_API_URL.replaceAll("\"", "");
```

Comment:
I think we don't need the `import.meta.env` part because we're defining the variable using the definePlugin from vite.

```suggestion
  axios.defaults.baseURL = VITE_API_URL.replaceAll("\"", "");
```

Reply:
interesting, I'll try that. Thanks

Reply:
nice, VITE_API_URL did the trick, and I could remove the replaceAll(..)